limitador-server/build.rs

```diff
@@ -1,4 +1,37 @@
+use std::process::Command;
+
 fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let git_hash = Command::new("git")
+        .args(&["rev-parse", "HEAD"])
+        .output()
+        .ok()
+        .filter(|output| output.status.success())
+        .and_then(|x| String::from_utf8(x.stdout).ok())
+        .map(|hash| hash[..8].to_owned());
+
+    let dirty = Command::new("git")
+        .args(&["diff", "--stat"])
+        .output()
+        .ok()
+        .filter(|output| output.status.success())
+        .map(|output| !matches!(output.stdout.len(), 0));
+
+    if Some(true) == dirty && git_hash.is_some() {
+        println!(
+            "cargo:rustc-env=LIMITADOR_GIT_HASH={}-dirty",
+            git_hash.unwrap_or_else(|| "unknown".to_owned())
```

Comment:
May be I should just call `git_hash.unwrap()` here, as there is no way this can be dirty from an "unknown" git tree...